test/e2e/serial/tests/resource_accounting.go

```diff
@@ -284,65 +291,80 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload resourc
+			baseload, err := nodes.GetLoad(fxt.K8sClient, targetNodeName)
+			ExpectWithOffset(1, err).ToNot(HaveOccurred(), "missing node load info for %q", targetNodeName)
+			By(fmt.Sprintf("computed base load: %s", baseload))
 
-			// add a min ammount of resources just in case all the zones
-			// are equal so the pod wont fit on any of them
-			excessRes := corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("100m"),
-				corev1.ResourceMemory: resource.MustParse("100M"),
+			//get maximum node CPU and Memory
+			maxResources := corev1.ResourceList{
+				corev1.ResourceCPU:    availableResourceType(*targetNodeNRTInitial, corev1.ResourceCPU),
+				corev1.ResourceMemory: availableResourceType(*targetNodeNRTInitial, corev1.ResourceMemory),
 			}
 			reqResources := maxResources.DeepCopy()
-
+			By("prepare the test's pod resources as maximum available resource on the target node without the baselaod")
+			//substract the baselaod from the max resources on the target node
 			reqCpu := reqResources[corev1.ResourceCPU]
-			reqCpu.Add(excessRes[corev1.ResourceCPU])
+			reqCpu.Sub(baseload.CPU)
 			reqResources[corev1.ResourceCPU] = reqCpu
-
 			reqMem := reqResources[corev1.ResourceMemory]
-			reqMem.Add(excessRes[corev1.ResourceMemory])
+			reqMem.Sub(baseload.Memory)
 			reqResources[corev1.ResourceMemory] = reqMem
+			By(fmt.Sprintf("test's pod requests: %s", e2ereslist.ToString(reqResources)))
 
-			By("Padding all other candidate nodes")
+			By("padding all other candidate nodes leaving room for the baseload on each node")
 			var paddingPods []*corev1.Pod
 			for _, nodeName := range nrtCandidateNames.List() {
 				node := &corev1.Node{}
 				nodeKey := client.ObjectKey{Name: nodeName}
-				err := fxt.Client.Get(context.TODO(), nodeKey, node)
+				err = fxt.Client.Get(context.TODO(), nodeKey, node)
 				Expect(err).NotTo(HaveOccurred())
 
-				// Padding all the other nodes until only maxResources is free
-				// remember reqResources = maxReources + excessResources
-				// so we ensure out pod will not be scheduled on any other node.
-				padPod, err := makeNodePaddingPod(fxt.Namespace.Name, *node, maxResources)
-				if errors.Is(err, e2enrt.ErrNotEnoughResources) {
-					klog.Infof("Node %q has not enough resources without padding", nodeName)
-					continue
+				//calculate base load on the node
+				baseload, err := nodes.GetLoad(fxt.K8sClient, nodeName)
+				ExpectWithOffset(1, err).ToNot(HaveOccurred(), "missing node load info for %q", nodeName)
+				By(fmt.Sprintf("computed base load: %s", baseload))
+
+				//get nrt info of the node
+				By(fmt.Sprintf("preparing node %q to fit the test case", nodeName))
+				nrtInfo, err := e2enrt.FindFromList(nrtCandidates, nodeName)
+				ExpectWithOffset(1, err).ToNot(HaveOccurred(), "missing NRT info for %q", nodeName)
+
+				//TODO: currently, the all the padding pods account for the total baseload, need to enhance distributing it on all numa zones equally (not sure if valid though)
```

Comment:
you can either account the baseload all on a (any) numa zone, or split it evenly. The former is probably simpler. I don't see atm a clearly better option.

Reply:
ok I'll go with the baseload on one numa because splitting may result in fractions of CPU that cannot be reflected in NRT. thanks for clarifying this point!

Reply:
@fromanirh  can we assume the base load of a node is not greater than the available topology on one numa? asking that to know if I need to tackle a case where the base load is not available on one numa

Reply:
if the detected baseload seems greater than the available resources on any NUMA zone, then the test should fail immediately, because it's very likely something is (very) wrong